Tools/setup/ubuntu.sh

```diff
@@ -145,7 +185,12 @@ if [[ $INSTALL_NUTTX == "true" ]]; then
 		u-boot-tools \
 		util-linux \
 		vim-common \
+		g++-arm-linux-gnueabihf \
+		gcc-arm-linux-gnueabihf \
+		g++-aarch64-linux-gnu \
```

Comment:
These aren't NuttX dependencies (the current section).

Reply:
Where would you move them to? the docker file?